go.mod

```diff
@@ -33,6 +33,6 @@ replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alp
 replace github.com/cosmos/ledger-cosmos-go => github.com/crypto-com/ledger-cosmos-go v0.9.10-0.20200929055312-01e1d341de0f
 
 // TODO: fix keyring upstream
-replace github.com/99designs/keyring => github.com/crypto-org-chain/keyring v1.1.6-gnome
+replace github.com/99designs/keyring => github.com/crypto-org-chain/keyring v1.1.6-fixes
```

Comment:
Is there a PR to merge these changes to upstream?

Reply:
yes, both from us and Regen: https://github.com/99designs/keyring/pulls